project_euler/problem_117/sol1.py

```diff
@@ -0,0 +1,52 @@
+"""
+Project Euler Problem 104: https://projecteuler.net/problem=117
```

Comment:
```suggestion
Project Euler Problem 117: https://projecteuler.net/problem=117
```
This seems like a typo.